docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -24,6 +24,85 @@ If Users use the same email address in Okta that they use with other authenticat
 
 To follow this guide, ensure you have permissions in your Okta workspace to create an OIDC app.
 
+## Create an Okta app
+
+1. From the Okta Admin Console, select **Create App Integration** from the **Applications** menu.
+1. For **Sign-in method**, select **OIDC - OpenID Connect**.
+1. For **Application type**, select **Web Application** and click **Next**.
+1. Configure **New Web App Integration Operations**:
+
+   - **App integration name**: Choose a name for the app.
+   - **Logo (optional)**: Add a logo.
+   - **Grant type**: Select **Authorization Code** and **Refresh Token**.
+   - **Sign-in redirect URIs**: Replace the default setting with the Grafana Cloud Okta path, replacing <YOUR_ORG> with the name of your Grafana organization: https://<YOUR_ORG>.grafana.net/login/okta. For on-premises installation, use the Grafana server URL: http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/okta.
+   - **Sign-out redirect URIs (optional)**: Replace the default setting with the Grafana Cloud Okta path, replacing <YOUR_ORG> with the name of your Grafana organization: https://<YOUR_ORG>.grafana.net/login/okta. For on-premises installation, use the Grafana server URL: http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/okta.
```

Comment:
I believe sign-out redirect URI is `http://<YOUR_ORG>.grafana.net/logout` or `http://<my_grafana_server_name_or_ip>:<grafana_server_port>/logout`

Reply:
I'll fix this.

Reply:
Fixed